implementation/src/main/java/io/smallrye/graphql/execution/datafetcher/helper/AbstractHelper.java

```diff
@@ -69,7 +69,7 @@ Object recursiveTransform(Object inputValue, Field field)
         } else if (Classes.isOptional(expectedType)) {
             // Also handle optionals
             return recursiveTransformOptional(inputValue, field);
-        } else if (shouldTransform(field)) {
+        } else {
```

Comment:
Why did you remove this check ?? (Just so I understand)

Reply:
Transformation is also used for types that have no TransformInfo (for example `UUID`), and instead of extending shouldTransform accordingly, the logic is contained in `Transformer#transform`. If no transformation is needed, `PassThroughTransformer` is used where no transformation is performed, so this check is unnecessary.

Reply:
Cool !